flang/lib/Lower/Bridge.cpp

```diff
@@ -819,6 +807,30 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     builder->create<fir::SelectOp>(loc, selectExpr, indexList, blockList);
   }
 
+  /// Collect DO CONCURRENT or FORALL loop control information.
+  IncrementLoopNestInfo getConcurrentControl(
+      const Fortran::parser::ConcurrentHeader &header,
+      const std::list<Fortran::parser::LocalitySpec> &localityList = {}) {
```

Comment:
I know this is from the front-end, but doesn't LLVM frown on using `std::list<T>`?

Reply:
No, there isn't.